geode-core/src/distributedTest/java/org/apache/geode/internal/cache/ClientDeserializationCopyOnReadRegressionTest.java

```diff
@@ -363,63 +332,61 @@ public void run2() throws CacheException {
         // All this extra work is to assure the serialization count does not increase
         // (by de-serializing the value stored in the map, which would then have to be
         // re-serialized).
-        pr.getDataStore().visitBuckets(new BucketVisitor() {
-          @Override
-          public void visit(Integer bucketId, Region r) {
-            BucketRegion br = (BucketRegion) r;
-            try {
-              KeyInfo keyInfo = new KeyInfo(k1, null, bucketId);
-              RawValue rv = br.getSerialized(keyInfo, false, false, null, null, false);
-              Object val = rv.getRawValue();
-              assertTrue(val instanceof CachedDeserializable);
-              CachedDeserializable cd = (CachedDeserializable) val;
-              SerializationCountingValue scv =
-                  (SerializationCountingValue) cd.getDeserializedForReading();
-              assertEquals(1, scv.count.get());
-            } catch (IOException fail) {
-              Assert.fail("Unexpected IOException", fail);
-            }
+        pr.getDataStore().visitBuckets((bucketId, r) -> {
+          BucketRegion br = (BucketRegion) r;
+          try {
+            KeyInfo keyInfo = new KeyInfo(k1, null, bucketId);
+            RawValue rv = br.getSerialized(keyInfo, false, false, null, null, false);
+            Object val = rv.getRawValue();
+            assertThat(val).isInstanceOf(CachedDeserializable.class);
+            CachedDeserializable cd = (CachedDeserializable) val;
+            SerializationCountingValue scv =
+                (SerializationCountingValue) cd.getDeserializedForReading();
+            assertThat(scv.count.get()).isEqualTo(1);
+          } catch (IOException fail) {
```

Comment:
If you convert the `new CacheSerializableRunnable() {` to a lambda, then you can delete the catch block which will be a lot cleaner.